src/hotspot/share/opto/loopTransform.cpp

```diff
@@ -2820,10 +2821,9 @@ void PhaseIdealLoop::do_range_check(IdealLoopTree *loop, Node_List &old_new) {
   assert(opqzm->outcnt() == 1, "cannot hack shared node");
   _igvn.replace_input_of(opqzm, 1, main_limit);
   // new main_limit can push Bool/Cmp nodes down (when one of the eliminated condition has parameters that are not loop
-  // invariant in the pre loop.
+  // invariant in the pre loop).
   set_ctrl(opqzm, new_limit_ctrl);
```

Comment:
Update this comment too.

Reply:
Both done in new commit. Do the comments look ok to you?